draft-ietf-rats-architecture.md

```diff
@@ -1458,27 +1486,23 @@ The following example illustrates a hypothetical Background-Check Model solution
         |                |                |               |
         ~                ~                ~               ~
         |                |                |               |
-     time(HR_a)<---------+-------------time(HR_v)------>time(HR_r)
+     time(HR_a)<----time(HT)-----------time(HR_v)----->time(HR_r)
```

Comment:
time(HT) is not used in any computation or inequality so should be removed for consistency

Reply:
I think the point was to provide more context by using a defined event in the text. I am rather cautious to agree with the notion that "used in a computation or inequality" is a needed requirement here. Also, it is used in a comparison/computation in the text.

We all agree that, for example, illustrating time(HR_a) in the diagram - although it cannot be captured by an Attester with no clock - is okay. Consequently, highlighting that time(HT) and time(HR_a) are related to effectively existing overlap phases although the delta is not easy to capture seems fine to me.

Reply:
I agree that illustrating time(HR_a) in the diagram is okay so long as it's used in a message or computation/inequality.